src/txn/txn_rollback_to_stable.c

```diff
@@ -1569,16 +1569,13 @@ __rollback_to_stable_hs_final_pass(WT_SESSION_IMPL *session, wt_timestamp_t roll
  *     Log a verbose message about the progress of the current rollback to stable.
  */
 static void
-__rollback_progress_msg(WT_SESSION_IMPL *session, struct timespec rollback_start,
-  uint64_t rollback_count, uint64_t *rollback_msg_count)
+__rollback_progress_msg(WT_SESSION_IMPL *session, WT_TIMER *rollback_start, uint64_t rollback_count,
+  uint64_t *rollback_msg_count)
 {
-    struct timespec cur_time;
     uint64_t time_diff;
 
-    __wt_epoch(session, &cur_time);
-
     /* Time since the rollback started. */
-    time_diff = WT_TIMEDIFF_SEC(cur_time, rollback_start);
+    __wt_timer_evaluate(session, rollback_start, &time_diff);
 
     if ((time_diff / WT_PROGRESS_MSG_PERIOD) > *rollback_msg_count) {
```

Comment:
There is a bug identified here and fixed in WT-11777. It is better to fix it with that ticket to avoid unnecessary extra logging.